_sass/_type.scss

```diff
@@ -125,16 +125,6 @@ h4 {
 
   a:link,
   a:visited {
-    display: inline;
-    text-decoration: underline;
-    text-decoration-color: lighten($orange, 25%);
-    text-decoration-thickness: 5px;
-    -webkit-text-decoration-skip: unset; 
-    text-decoration-skip: unset; 
-    text-decoration-skip-ink: none;
-
-    &::before {
-      display: none;
-    }
+    --link-bg-color: #{lighten($orange, 25%)};
```

Comment:
I don't think we need this in the `awards-list` now. It's looking more creme-scicle than the actual orange. Feel free to blow away any special styling for the awards list.

Reply:
Agreed